### PR TITLE
Iac userstory

### PR DESCRIPTION
**Pull request by wereinse**
## Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

### Validation
- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

### Issues Closed or Referenced

- References https://github.com/retaildevcrews/helium-iac/issues/33

### CODE REVIEW

File: docs/HeliumIacUserStory.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Helium IAC User Story
+
+## User Stories
+
+- As a DevOps engineer, I want a repeatable way to deploy infrastructure as code via CI-CD that is secure by default and follows engineering best practices.
```
Comment (dsturgell) [inline docs/HeliumIacUserStory.md:5]: Need to determine the right audience - needs to be specific to a persona

File: docs/HeliumIacUserStory.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+- Security
+  - "Secure by Design"
+  - Key Vaults
+  - Developer Experience
+
+- Cloud Native Platform (containerization)
+  - ACR
+  - App Services
+  - AKS
```
Comment (dsturgell) [inline docs/HeliumIacUserStory.md:33]: Does the terraform stand up an AKS instance?  If not, should be removed.

File: docs/HeliumIacUserStory.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  - DevOps
+  - IaC Standard components
+  - CI-CD validation
+  - Application Insights
+  - Health Checks
+  - Alerting
+  - Dashboards
+
+- CosmosDB Infrastructure
+  - Document Modeling
```
Comment (dsturgell) [inline docs/HeliumIacUserStory.md:43]: Recommend removing as this applies to Helium/IMDb.

File: docs/HeliumIacUserStory.md (hunk 1 above)
Comment (wereinse) [reply docs/HeliumIacUserStory.md:5]: Ok - I struggled with that.  Developers could use it and then Ops personnel who only have access to stage or prod may use it.

File: docs/HeliumIacUserStory.md (hunk 2 above)
Comment (wereinse) [reply docs/HeliumIacUserStory.md:33]: It doesn't thank you for catching.  I'll remove that.

**Pull request merged**